documentation/schema/Syriacamsdesc.odd

```diff
@@ -4440,8 +4627,87 @@
                         </div>
                     </div>
                     <div xml:id="handdesc">
-                        <head>Hand Description [UNDER REVISION]</head>
-                        <p>Further documentation will be added here.</p>
+                        <head>Hand Description</head>
+                        <p>Each distinct hand and/or script in the codicological unit has been
+                            described in a separate <gi>handNote</gi> element in the
+                            <gi>handDesc</gi> element. The <gi>handDesc</gi> element itself
+                            has an attribute <att>hands</att> which has been used to record the
+                            number of hands identified, including both those involved in the
+                            original production of the unit and those contributing later additions.</p>
+                        <p>Each <gi>handNote</gi> element must have an <att>xml:id</att> attribute.</p>
+                        <p>The <att>scope</att> attribute can be used to indicate the extent of
+                            the hand's appearance in the manuscript. Its values can be one
+                            of:</p>
+                        <list>
+                            <item>
+                                <val>sole</val>
+                            </item>
+                            <item>
+                                <val>major</val>
+                            </item>
+                            <item>
+                                <val>minor</val>
+                            </item>
+                        </list>
+                        <p>The <att>script</att> attribute provides a
+                            classification of the script. Oftentimes, only the broadest
+                            classification is possible. <title>SMBL</title> has used a combination
+                            of an ISO 639 Language Code with an optional <ref target="https://www.unicode.org/iso15924/">ISO 15924</ref> script code to encode a hand's script information. The <att>script</att> can take multiple, space-separated script values</p>
+                        <p>The accepted script values are:</p>
+                        <table>
+                            <row><cell>ar</cell><cell>Unspecified Arabic script</cell></row>
+                            <row><cell>ar-Syrc</cell><cell>Arabic Garshuni in Unspecified Syriac script</cell></row>
+                            <row><cell>ar-Syrn</cell><cell>Arabic Garshuni in East Syriac script</cell></row>
```

Comment:
@wlpotter We do not have this in the list above. Should we add it?

Reply:
The difference here is that these are specific `@script` values, not all of which should be used for `@xml:lang` or `@mainLang` attributes

Reply:
Syro-Arabic Garshunography in East Syriac script

Reply:
Syro-Arabic Garshunography in unspecified Syriac script